### PR TITLE
108 auto email confirmation

### PR DESCRIPTION
**Pull request by jameschou93**
[Trello Card](https://trello.com/c/LYOIdSTR/108-automatic-email-sent-to-the-person-contacting-imti-to-confirm-that-imti-received-the-email)

### CODE REVIEW

File: app/helpers/pages_helper.rb [hunk 1]
```diff
@@ -18,5 +17,57 @@ def send_email_mailgun
...
+    p "MESSAGE_ID IS :    #{message_id}"
+    mes_ids = []
+    json['items'].each do |item|
+        mes_ids << item['message']['headers']['message-id']
+        # message_ids << item['message']['headers']['message-id']
+    end
+    mes_ids.each do |mes|
+      p "mes is ::::"
+      p mes
+    end
```
Comment (trevordjones) [inline app/helpers/pages_helper.rb]: Do we need all this stuff?

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -18,11 +18,17 @@ def contact
   end
 
   def send_message
-    send_email_mailgun
+    if send_email_mailgun
```
Comment (trevordjones) [inline app/controllers/pages_controller.rb]: Will this always return something truthy? Ever return anything falsey? (nil, false)

File: app/helpers/pages_helper.rb (hunk 1 above)
Comment (jameschou93) [reply app/helpers/pages_helper.rb]: will delete

File: app/controllers/pages_controller.rb (hunk 2 above)
Comment (jameschou93) [reply app/controllers/pages_controller.rb]: I'll change it back

**Comment by jameschou93** [comment]
Changes made

**Pull request merged**